libecl/src/ecl_kw.c

```diff
@@ -2518,36 +2518,39 @@ bool ecl_kw_is_kw_file(fortio_type * fortio) {
 }
 
 
-
-
+static bool ecl_kw_is_grdecl_file_rewind(FILE * stream, const long int pos, bool ret_val) {
```

Comment:
Since this function is so *very* internal, I think it should not even get the `ecl_kw_`prefix - I would prefer something like:
`rewind_and_return()`?